### PR TITLE
update to latest renv

### PR DESCRIPTION
**Pull request by kevinushey**
Hopefully better luck with code coverage?

### CODE REVIEW

File: tests/testthat/helper.R [hunk 1]
```diff
@@ -26,3 +26,13 @@ expect_api_routes <- function(routes) {
...
+test_that <- function(desc, code) {
+    writeLines("")
+    writeLines(renv$header(paste(desc, "[begin]")))
+    call <- sys.call()
+    call[[1]] <- quote(testthat::test_that)
+    eval(call, envir = parent.frame())
+    writeLines(renv$header(paste(desc, "[ended]")))
+    writeLines("")
+    writeLines("")
+}
```
Comment (juliasilge) [inline tests/testthat/helper.R:38]: Is this still needed? Or was it only for debugging?

**Comment by juliasilge** [comment]
Thank you so much for working on this! 🙏

File: tests/testthat/helper.R (hunk 1 above)
Comment (kevinushey) [reply tests/testthat/helper.R:38]: Just for debugging :-) I'll clean up the PR.

**Comment by kevinushey** [comment]
Should now be ready to go!

**Comment by juliasilge** [comment]
Closed in #218, the older PR with some related changes that I just updated again.

Thank you again so much!

**Pull request closed without merging**